src/subscription_manager/gui/about.py

```diff
@@ -73,10 +73,10 @@ def __init__(self, parent, backend):
         server_versions = get_server_versions(self.backend.cp_provider.get_consumer_auth_cp())
         client_versions = get_client_versions()
 
-        sub_man_version_label.set_markup(_("<b>%s version:</b> %s") %
-                                        ("subscription manager", client_versions['subscription-manager']))
-        backend_version_label.set_markup(_("<b>subscription management service version:</b> %s") %
-                                           server_versions['candlepin'])
+        sub_man_version_label.set_markup(_("<b>{name} version:</b> {version}").format(
+            name="subscription manager", version=client_versions['subscription-manager']))
+        backend_version_label.set_markup(_("<b>subscription management service {version}:</b> {version}").format(
```

Comment:
the first `version` is just a word, no need to replace it

Reply:
I think I was thinking I forgot the {} around it.  changed it back